ionos_common/grafana-dashboards/loki-basic-apps-dashboard.yaml

```diff
@@ -0,0 +1,292 @@
+# apiVersion: v1
+# kind: ConfigMap
```

Comment:
Is there any reason this is commented out? Maybe it's better noot to have this file at all?

Reply:
Please do the rebase

Reply:
removed the file, I did not notice it